shell/config/router/navigation-guards/history.js

```diff
@@ -0,0 +1,13 @@
+export function install(router, context) {
+  router.beforeEach((to, from, next) => loadHistory(to, from, next, context));
+}
+
+export async function loadHistory(to, from, next) {
+  // Clear state used to record if back button was used for navigation
+  // TODO: Investigate if this can be removed. This is only used on the templates/error.vue page and seems hacky.
```

Comment:
Do we still use the `template/error` page?
Because I think it was just on initialization.
Should we maybe open an issue to make an investigation in general on how is handled the 404, so we could merge all to `failWhale` instead?

Reply:
I opened two issues, one for this todo and one for the requests 404/failwhale investigation.

https://github.com/rancher/dashboard/issues/11384

https://github.com/rancher/dashboard/issues/11385

Reply:
Thank you! 🍺